clients/gax/test/gax/request_test.exs

```diff
@@ -59,10 +59,10 @@ defmodule Gax.RequestTest do
   test "adds parameter" do
     request =
       Request.new()
-      |> Request.add_param(:query, :foo, 'asdf')
```

Comment:
Why were these lists previously? (Is there the possibility that users might pass in a char list for this argument, expecting it to be treated as a such?)

Reply:
These were accidentally put in as charlists. They are documented as `String.t` and in general, I expect Elixir users to want to deal with Strings over char lists.